arcgis-ios-sdk-samples/Utility network/Perform valve isolation trace/PerformValveIsolationTraceViewController.swift

```diff
@@ -104,6 +91,31 @@ class PerformValveIsolationTraceViewController: UIViewController {
         return parameters
     }
     
+    /// Load the service geodatabase and initialize the layers.
+    func loadServiceGeodatabase() {
+        serviceGeodatabase = AGSServiceGeodatabase(url: featureServiceURL)
+        serviceGeodatabase.load { [ weak self ] error in
```

Comment:
```suggestion
        serviceGeodatabase.load { [weak self] error in
```
Also what about moving the error checking to the top of this load method? It seems odd to check error at the end. Though I'm not sure what error would be emitted other than network or load error.